st2common/st2common/util/monkey_patch.py

```diff
@@ -31,7 +31,7 @@
 PARENT_ARGS_FLAG = "--parent-args="
 
 
-def monkey_patch(patch_thread=None):
+def monkey_patch(patch_thread=False):
```

Comment:
Just curious how what ``patch_thread=False`` affects things?

I assume it will cause issues with blocking the event loop if we have some code which utilizes threading module.